### DISCUSSION TITLE
[Feature] time and date scaling options

### DISCUSSION THREAD

**Discussion by lobs323** [damongolding/immich-kiosk#86] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I've just built a pi Immich frame using and old 4k laptop screen and the metadata size on the screen for dates etc are absolutely tiny. 

I'm not sure how you've set the size as it is, but would it be possible to allow modification of the size by percentage where 100% is the default size?

Otherwise just wanted to say I love this project! Thank you.

**Comment by damongolding** [comment]
All of the font are set as REMs, so as a percentage of the base font size (which is normally 16px). 

Would you be looking to change the base font size? So all the UI forms would scale accordingly? Or more granular alterations?

Another thing to note is that if you are able to change the font size via a keyboard (cmd/ctrl +) then Kiosk will respect and scale.

**Reply by lobs323** [reply]
Oh I hadn't thought to change it as scaling on the device itself 😅 I'll try and figure out how to do it via ssh since it's currently in a picture frame on my wall!

**Reply by lobs323** [reply]
xrandr with --scale was the answer!

**Reply by damongolding** [reply]
Are you using chromium? You could set the base font size using this flag as a short term solution. 

```
chromium --force-device-scale-factor=1 --default-font-size=18
```

Which would set the base font size to 18px (the default is normally 16px)

**Comment by lobs323** [comment]
![PXL_20240914_171429386](https://github.com/user-attachments/assets/9ddc0afc-d0fe-409d-a2fb-2093303a5bdb)

Scaling allowed me to see the clock, but the image dates have been pushed off the screen now 😕

**Comment by damongolding** [comment]
That’s a shame but I was going to implement font size adjustment or custom CSS at some point so I might bake it into 0.9.1 as I need to release a fix for raw images.

**Comment by damongolding** [comment]
@lobs323 if you fancy being a beta tester you could try out the `damongolding/immich-kiosk-development:0.10.0` image which has a `font_size` (config and url param) or `KIOSK_FONT_SIZE` (ENV) option, which is a int (default is 100). e.g. `http://{KIOSK_URL}?font_size=200` would set the base font size to 200% (32px) etc

If you so could you add any issues/comments here please 🙏

**Reply by soana** [reply]
Nice addition - works fine for me.

**Comment by damongolding** [comment]
Added in v0.10.0
